devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
Two changes:
 - you moved `make PythonInstall` before `make install` in https://github.com/peastman/openmm/commit/46fb5c511121a2df7921b013972fe16b730b1d74, which is wrong. 
 - And in <7.0, `make PythonApiDocs` didn't require `make PythonInstall.